pkg/cmd/server/start/start_master.go

```diff
@@ -754,7 +754,6 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 
 	// no special order
 	if configapi.IsBuildEnabled(&oc.Options) {
-		oc.RunBuildPodController()
```

Comment:
i'd like to understand why this block is special, instead of handling the buildconfigchangecontroller in the for loop above.